src/app/[locale]/incident/add/components/questions/LocationSelect.tsx

```diff
@@ -72,9 +72,14 @@ export const LocationSelect = ({ field }: LocationSelectProps) => {
                   {t('choose_location')}
                 </Button>
               ) : (
-                <LinkButton inline={true} id="location-button" type="button">
+                <Button
```

Comment:
Volgens mij bedoelde @Robbert in #229 dat deze button in een`<ButtonGroup>` Component geplaatst kan worden?
(Zie [Utrecht Storybook](https://nl-design-system.github.io/utrecht/storybook-react/?path=/docs/react-button-group--docs))

Reply:
Ik snap niet zo goed het nut van een ButtonGroup in het geval van enkel het tonen van een button, er zijn er twee maar ze worden nooit samen getoond.

Reply:
Fair point, ik denk (maar not sure) dat ButtonGroup ook voor 1 button gebruikt kan worden. Ik zou dan verwachten dat dit het geval is als 1 Button zowel de "Kies locatie" als "Wijzig locatie" functie doet. Als dit aparte buttons zijn dan weet ik ook niet of het zin heeft om eentje daarvan in een ButtonGroup te plaatsen 🤔 

@Robbert ?